packages/EasyStandard/src/Sniffs/Arrays/OrderArrayKeysAlphabeticallySniff.php

```diff
@@ -0,0 +1,253 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyStandard\Sniffs\Arrays;
+
+use Error;
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PhpParser\Comment;
+use PhpParser\Node\Expr\Array_;
+use PhpParser\Node\Expr\ArrayItem;
+use PhpParser\ParserFactory;
+use PhpParser\PrettyPrinter\Standard;
+use SlevomatCodingStandard\Helpers\TokenHelper;
+
+final class OrderArrayKeysAlphabeticallySniff implements Sniff
+{
+    /**
+     * @var string
+     */
+    public const ARRAY_KEYS_SORT_ALPHABETICALLY = 'ArrayKeysSortAlphabetically';
+
+    /**
+     * @var string
+     */
+    private const COMMENT_CONTENT = '[comment]';
```

Comment:
Let's replace it with `TEMP_COMMENT_CONTENT = '[temp comment]'` and add the following description:
```
This comment is temporarily added to the lines of a multiline array to keep it multiline after fixing the order.
```

Reply:
Done.